src/key.h

```diff
@@ -78,6 +78,7 @@ static inline void scoutfs_key_set_zeros(struct scoutfs_key *key)
 	key->_sk_second = 0;
 	key->_sk_third = 0;
 	key->_sk_fourth = 0;
+	memset(key->__pad, 0, ARRAY_SIZE(key->__pad));
```

Comment:
That'd be wrong if someone didn't use u8 __pad types, just use sizeof() instead of ARRAY_SIZE().